.editorconfig

```diff
@@ -0,0 +1,9 @@
+[*]
+indent_style = space
+end_of_line = lf
```

Comment:
Git for Windows converts the files to CRLF automagically, so even though they are committed as LF, cloning the repository on Windows will actually give you CRLF.

```suggestion
```